pkg/auth.go

```diff
@@ -321,7 +321,22 @@ func (auth *Auth) CreateAccessToken(request services.CreateAccessTokenRequest) (
 
 // CreateJwtBearerToken creates a JwtBearerToken from the given CreateJwtBearerTokenRequest
 func (auth *Auth) CreateJwtBearerToken(request services.CreateJwtBearerTokenRequest) (*services.JwtBearerTokenResult, error) {
-	return auth.AccessTokenHandler.CreateJwtBearerToken(&request)
+	// todo add checks?
+	custodian, err := core.ParsePartyID(request.Custodian)
+	if err != nil {
+		return nil, err
+	}
+	endpointType := "oauth" // todo
```

Comment:
is this dependent on the services we're introducing?

Reply:
yes, also when the relation between scope and service is further speced, the entire func will have to find the endpoint that matches the service that matches the scope.